PointsOfInterest/atm.csv

```diff
@@ -0,0 +1,9 @@
+name,description,note,latitude,longitude
+CIBC,,Student Life Centre (SLC),43.47160888938205,-80.54564427965785
+CIBC,,Student Life Centre (SLC),43.471581,-80.545526
+CIBC,,William G. Davis Computer Research Centre (DC),43.47298766617178,-80.54265104234219

```

Comment:
As per the [last](https://github.com/uWaterloo/Datasets/pull/157#discussion_r49494970) comment... isn't this redundant?